loadtest.go

```diff
@@ -103,13 +104,31 @@ func (l *loadTest) runSubscriber(ctx context.Context, client Client, userNum int
 			for {
 				l.log.Debug("subscribing", "channel", channel)
 				err := client.Subscribe(ctx, channel, func(data []byte) {
-					var msg Message
-					if err := json.Unmarshal(data, &msg); err != nil {
-						l.log.Debug("error parsing message", "err", err)
-						l.w.boomer.RecordFailure("ablyboomer", "subscribe", 0, err.Error())
-						return
+					var latency int64
+					if l.w.Conf().Subscriber.Push {
+						l.log.Debug("subscriber received message", "message", string(data))
+						type pushData struct {
+							Time int64
+						}
+						type pushMessage struct {
+							Data pushData
+						}
+						var msg pushMessage
```

Comment:
Should we perhaps make the non-push message have the same format as the push-message to avoid this conditional?